CMake Error at tests/unit-tests/CMakeLists.txt:43 (add_dependencies):
  The dependency target "GMock" of target "lsc_unit_tests" does not exist.


CMake Error at tests/integration-tests/CMakeLists.txt:57 (add_dependencies):
  The dependency target "GMock" of target "lsc_integration_tests" does not
  exist.


diff --git a/tests/integration-tests/CMakeLists.txt b/tests/integration-tests/CMakeLists.txt
index eb54ce3..a0b6fb3 100644
--- a/tests/integration-tests/CMakeLists.txt
+++ b/tests/integration-tests/CMakeLists.txt
@@ -54,7 +54,7 @@ target_link_libraries(
 
 add_test(lsc_integration_tests ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/lsc_integration_tests)
 
-add_dependencies(lsc_integration_tests GMock)
+add_dependencies(lsc_integration_tests ${GMOCK_LIBRARIES})
 add_dependencies(lsc_integration_tests lsc_test_helper_wait_for_signal)
 
 install(TARGETS lsc_integration_tests lsc_test_helper_wait_for_signal
diff --git a/tests/unit-tests/CMakeLists.txt b/tests/unit-tests/CMakeLists.txt
index 5307f8a..ac40555 100644
--- a/tests/unit-tests/CMakeLists.txt
+++ b/tests/unit-tests/CMakeLists.txt
@@ -40,4 +40,4 @@ target_link_libraries(
 
 add_test(lsc_unit_tests ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/lsc_unit_tests)
 
-add_dependencies(lsc_unit_tests GMock)
+add_dependencies(lsc_unit_tests ${GMOCK_LIBRARIES})
